rethinkdb/datadog_checks/rethinkdb/config.py

```diff
@@ -0,0 +1,55 @@
+# (C) Datadog, Inc. 2020-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+from typing import List, Optional
+
+from datadog_checks.base import ConfigurationError
+
+from .types import Instance
+
+
+class Config(object):
+    """
+    Hold instance configuration for a RethinkDB check.
+
+    Encapsulates the validation of an `instance` dictionary while improving type information.
+    """
+
+    def __init__(self, instance=None):
+        # type: (Instance) -> None
+        if instance is None:
+            instance = {}
+
+        host = instance.get('host', 'localhost')
+        port = instance.get('port', 28015)
```

Comment:
In case they quote it

```suggestion
        port = int(instance.get('port', 28015))
```